Added new assembly type table.

mscheremetjew opened the pull request.

mscheremetjew committed:
Added new table assembly type.

mberacochea commented on backlog/models.py:
```diff
@@ -141,6 +141,17 @@ class Meta:
+
+
 # Assemblies received from ENA
 class Assembly(models.Model):
```
This model has several FKs (they are lazy loaded by default). My guess is that all that info will be loaded and used most of the time, to improve the performance I would add a custom Manager like this:

```python
class AssemblyManager(models.Manager):

    def get_queryset(self):
        qs = super().get_queryset()
        return qs.select_related('study', 'biome', 'inferred_biome', 'assembly_type')
```

mberacochea committed:
Annotations Jobs pipeline results status

This field will allow us to record the pipeline results status.
Such as "Failed QC" or "No CDS"

Vkale1 committed:
Merge pull request #22 from EBI-Metagenomics/annotation-pipeline-status

Annotations Jobs pipeline results status
